src/grammar/textToSan

```diff
@@ -82,7 +82,7 @@ en_passant
                   final_rank = 3;
                   break;
               case '5':           // white capturing black
-                  final_rank = 4;
+                  final_rank = 6;
```

Comment:
Good catch! Can you please put this en passant fix in a separate pull request? That way I can merge it immediately while reviewing the more extensive changes for supporting multiple languages.